docs/operational_guide/availability_consistency_durability.md

```diff
@@ -0,0 +1,71 @@
+
+We recommend referring to [this sample M3 YAML configuration file](https://github.com/m3db/m3/blob/master/src/dbnode/config/m3dbnode-all-config.yml) which has every possible configuration value set while reading the rest of this document to see how the described configurations fit into the YAML file as a whole.
+
+### Client Write and Read Consistency
+
+The possible configuration values for write and read consistency are discussed in more detail [in this section](../m3db/architecture/consistencylevels.md) of the documentation, but in short M3DB behaves similarly to other H.A systems with configurable consistency such as Cassandra that allow the caller to control the consistency level of writes and reads from the client.
+
+The most important thing to understand is that if you want the guarantee that you will be able to read the outcome of every succesful write then you must perform both writes and read with a consistency level of `majority`.
+
+### Commitlog Configuration
+
+By default M3DB runs with an asynchronous commitlog such that writes will be acknowleged as successful by the client even though the data may not have been physically flushed to the commitlog on disk yet. M3DB supports changing this default behavior to run the commitlog synchronously, but this is not currently exposed to users in the YAML configuration and generally leads to a massive performance degradation.
+
+M3DB operators can control when M3DB will attempt to flush the commitlog via the `commitlog` section of the YAML configuration. For example the default configuration:
+
+```
+commitlog:
+	flushMaxBytes: 524288
+	flushEvery: 1s
+	queue:
+			calculationType: fixed
+			size: 2097152
+```
+
+states that the commitlog should be flushed whenever one of the following is true:
+
+1. 524288 or more bytes have been written since the last time M3DB flushed the commitlog.
+2. One or more seconds has elapsed since the last time M3DB flushed the commitlog.
+
+In addition, the configuration also states that M3DB should allow up to `2097152` writes to be buffered in the commitlog queue before the database node will begin rejecting incoming writes so it can attempt to drain the queue and catch up. Increasing the size of this queue can often increase the write throughput of an M3DB node at the cost of potentially losing more data if the node experiences a sudden failure like a hard crash or power loss.
+
+### Writing New Series Asynchronously
+
+The default M3DB YAML configuration will contain the following as a top-level key under the `db` section:
```

Comment:
nit:

> The [default M3DB configuration](https://github.com/m3db/m3/blob/master/src/dbnode/config/m3dbnode-all-config.yml) contains the following [...]

Reply:
this is tricky to link to because we have multiple "defaults" depending on whether coordinator/etcd are running embedded etc

Reply:
ah, true - similar to gearing the docs/defaults to obs-centric use cases, it might be good to assume a primary mode of operation as well (probably not an embedded one).

no issue with keeping it generalized though, in which case we should prob just link to the dir instead so that folks can reference the file that is most relevant to their use case.